query-engine/prisma-models/src/model.rs

```diff
@@ -88,6 +106,7 @@ impl Model {
 impl Debug for Model {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         f.debug_struct("Model")
+            .field("schema", &self.schema_name().unwrap_or_default())
             .field("name", &self.name)
```

Comment:
Would you mind switching those? It's a detail but I'd like to keep the model name first